README.md

```diff
@@ -1,2 +1,54 @@
 # commercetools-cli-scripts
 Collection of node.js based scripts and utils for CTP resource management
+
+## Requirements
+Download Node.js installer
+https://nodejs.org/en/download/
+
+## Setup
+Inside /features/shared folder, input following command in terminal to install required javascript library
```

Comment:
Here you need to give one command to install all you don't need to specify every dependency. Less description is more ready to go commands. Following the current documentation is hard. This is why I said before using one package.json.

Reply:
https://github.com/commercetools/commercetools-cli-scripts/commit/d7e9a3b2f9074b75fbb082811ccd724c264a610b